corehq/messaging/scheduling/forms.py

```diff
@@ -2683,6 +2711,9 @@ class ConditionalAlertScheduleForm(ScheduleForm):
     START_OFFSET_NEGATIVE = 'NEGATIVE'
     START_OFFSET_POSITIVE = 'POSITIVE'
 
+    MIN_SMS_STALE_INTERVAL = 0
```

Comment:
can we add a comment telling what this is?
`MIN_SMS_STALE_INTERVAL = 0  #days` or hours depending on what you finalize